packages/contracts/test/vbridge/signatureVBridge.test.ts

```diff
@@ -1009,7 +1009,7 @@ describe.skip('2-sided multichain tests for signature vbridge', () => {
   });
 });
 
-describe('8-sided multichain tests for signature vbridge', () => {
+describe.skip('8-sided multichain tests for signature vbridge', () => {
```

Comment:
locally this and the 2-sided bridge work for me, I think we can remove the skips from this file